src/Renderer/PointsMaterial.js

```diff
@@ -26,19 +28,31 @@ class PointsMaterial extends RawShaderMaterial {
         if (__DEBUG__) {
             this.defines.DEBUG = 1;
         }
-
         this.updateUniforms();
     }
 
     enablePicking(pickingMode) {
         // we don't want pixels to blend over already drawn pixels
-        this.blending = pickingMode ? NoBlending : NormalBlending;
         this.uniforms.pickingMode.value = pickingMode;
+        this.blending = pickingMode ? NoBlending : NormalBlending;
     }
 
     updateUniforms() {
         // if size is null, switch to autosizing using the canvas height
         this.uniforms.size.value = (this.size > 0) ? this.size : -this.scale * window.innerHeight;
+        this.uniforms.opacity.value = this.opacity;
+        this.uniforms.overlayColor.value = this.overlayColor;
+    }
+
+    copy(source) {
+        this.visible = source.visible;
+        this.opacity = source.opacity;
+        this.transparent = source.transparent;
+        this.size = source.size;
+        this.scale = source.scale;
+        this.overlayColor.copy(source.overlayColor);
+        this.updateUniforms();
```

Comment:
Another too late comment: you're overriding three.js `copy` method but you don't call the `super` method.
This should be fixed:
- either rename your `copy` method and keep its current code
- keep the `copy` name and call RawShaderMaterial.copy as well

Reply:
I have just tried proper overriding, but it does not work (unless you manually set `this.uniforms = source.uniforms` after the call to `super`). I do not know if it is a misuse or an upstream bug, so meanwhile, adding an optional `source` parameter to `updateUniforms` could do the trick.